tests/models/test_horovod.py

```diff
@@ -155,7 +146,7 @@ def test_horovod_apex(tmpdir):
 
 @pytest.mark.skip(reason="Skip till Horovod fixes integration with Native torch.cuda.amp")
```

Comment:
This test is working now I now. We can remove `@pytest.mark.skip(reason="Skip till Horovod fixes integration with Native torch.cuda.amp")`

Reply:
seems to fail on Drone, so let it add on separate PR